#26 feature/calculation parameters screen

Lucas-Dreveck opened the pull request:
Closes #26 

## 📑 Description
Adicionada a página de configurações usando a rota `settings` onde tem um formúlario com todos os dados dos `calculation_parameters` para alteração livremente seguindo as regras de negócio e com as validações apropriadas.

## Additional Information
Durante a task também analisei e reservei problemas ou questões a resolver para padronizar o front. Irei criar uma branch e uma issue a partir desta para realizar todas essas soluções.

Lucas-Dreveck committed:
refactor(web): removed unused utils files and added Readonly to main `layout.tsx`

Removed all utils files because of not being used in all the project and change the children of the
main `layout.tsx` to Readonly to improve performance

Lucas-Dreveck committed:
feat(web): create settings page with all calculation parameters editable

added the settings screen with a form to edit all the calculation parameters fetched from the server

#26

heloshartmann reviewed (changes requested):
Esta padrão o 0 dai não tem como apagar ele, arrumar isso.
Erro ao tentar salvar novos parâmetros.

Lucas-Dreveck committed:
Merge branch 'development' into feature/26-calculation-parameters-screen

Lucas-Dreveck committed:
refactor(web): updated the default value and changed the fields for number only

changed from default 0 to default '' and make all the fields only accept numbers with both english
and portuguese decimal separators

Lucas-Dreveck commented:
> Esta padrão o 0 dai não tem como apagar ele, arrumar isso. Erro ao tentar salvar novos parâmetros.

Acredito estar tudo certo agora, irei apenas arrumar o problema da vulnerabilidade em outra branch para puxar para todas as que estão sendo barradas atualmente

Lucas-Dreveck committed:
Merge branch 'development' into feature/26-calculation-parameters-screen
